app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala

```diff
@@ -84,19 +84,19 @@ class ReminderController @Inject() (
   def getEmailsavingsReminders(): Action[AnyContent] =
     authorisedForHtsWithNINO { implicit request ⇒ implicit htsContext ⇒
       if (isFeatureEnabled) {
-        // get optin status
+        // get optin statu
```

Comment:
small change..please correct it

Reply:
Done, but the typo is itself due to mispalced mouse cursor, if I am right.